src/backend/joanie/payment/backends/lyra/__init__.py

```diff
@@ -129,26 +129,27 @@ def _call_api(self, url, payload):
                 e,
                 extra={"context": context},
             )
-            return None
+            raise exceptions.PaymentProviderAPIException(
```

Comment:
As you are now raising an exception, I wonder if you should not transform the `logger.error` above into `logger.info`.

@kernicPanel WDYT ?

Reply:
Calling logger.error and raising an uncatched exception would open two issues on sentry, I think.
We should give it a try locallyx

Reply:
How can I test this with sentry ?

Reply:
On sentry's side, I have only 1 event that was listed. It seems like sentry will catch the `logger.error` only.
On the application side error, we will have the `logger.error` message and then the `exception` message.

Reply:
It raises 1 event on Sentry.